perfdash/parser.go

```diff
@@ -291,6 +291,7 @@ func parseSchedulingLatency(testName string) func([]byte, int, *BuildData) {
 
 type schedulingThroughputMetric struct {
 	Average float64 `json:"average"`
+	Max float64 `json:"max"`
```

Comment:
Formatting? I think that all "float64" should be in the same column. Running gofmt on this file should do the trick.